### PR TITLE
Print non-report output to stdErr

### PR DESCRIPTION
**Pull request by keradus**
Appendix for #1824

### CODE REVIEW

File: src/Console/Command/FixCommand.php [hunk 1]
```diff
@@ -318,10 +315,24 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
         ;
 
+        $reporter = ReporterFactory::create()
+            ->registerBuiltInReporters()
+            ->getReporter($resolver->getFormat())
+        ;
+
+        $stdErr = $output instanceof ConsoleOutputInterface
+            ? $output->getErrorOutput()
+            : ('txt' === $reporter->getFormat() ? $output : null)
```
Comment (GrahamCampbell) [inline src/Console/Command/FixCommand.php]: I find nested ternary statements hard to read, and can be avoided.

Comment (GrahamCampbell) [inline src/Console/Command/FixCommand.php]: If it doesn't fit on one line, i'd always go for plain old if statements.

File: src/Console/Command/FixCommand.php [hunk 2]
```diff
@@ -320,7 +320,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
             ->getReporter($resolver->getFormat())
         ;
 
-        $stdErr = ($output instanceof ConsoleOutputInterface) ? $output->getErrorOutput() : 'txt' === $reporter->getFormat() ? $output : null;
+        $stdErr = $output instanceof ConsoleOutputInterface
+            ? $output->getErrorOutput()
+            : ('txt' === $reporter->getFormat() ? $output : null)
+        ;
```
Comment (keradus) [inline src/Console/Command/FixCommand.php]: @SpacePossum 

```php
($output instanceof ConsoleOutputInterface) ? $output->getErrorOutput() : 'txt' === $reporter->getFormat() ? $output : null;
```
means
```php
(($output instanceof ConsoleOutputInterface) ? $output->getErrorOutput() : 'txt' === $reporter->getFormat()) ? $output : null;
```
not
```php
($output instanceof ConsoleOutputInterface) ? $output->getErrorOutput() : ('txt' === $reporter->getFormat() ? $output : null);
```

-------------------

before my change this always returns `$output`

Comment (GrahamCampbell) [inline src/Console/Command/FixCommand.php]: good enough reason not to use nested ternary statements then :P

Comment (GrahamCampbell) [inline src/Console/Command/FixCommand.php]: people have already got confused

File: src/Console/Command/FixCommand.php (hunk 1 above)
Comment (keradus) [inline src/Console/Command/FixCommand.php]: I find nested ternary statements nice to read as long as they are properly used.

File: src/Console/Command/FixCommand.php (hunk 2 above)
Comment (keradus) [inline src/Console/Command/FixCommand.php]: @GrahamCampbell , this PR is only for allowing me merging #1824 without waiting another week. I will merge it as soon as CI will be green.

Comment (GrahamCampbell) [inline src/Console/Command/FixCommand.php]: well, it's replacing something broken with something also broken

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request merged**